src/main/java/org/osc/manager/ism/entities/PolicyElements.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.ism.entities;
+
+
+
+
+public class PolicyElements {
```

Comment:
I agree. For the most part, the model classes might go away. We might still need some model classes to be present in case there is no direct mapping possible between the *Element interfaces and the entities.